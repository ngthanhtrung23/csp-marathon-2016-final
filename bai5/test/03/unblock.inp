12233.
1.4...
00456.
...56.
77756.
......
