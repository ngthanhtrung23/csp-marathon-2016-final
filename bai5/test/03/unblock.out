15
5 U 1
6 U 1
7 R 3
4 D 2
0 R 1
1 D 3
0 L 1
2 L 1
4 U 3
0 R 1
1 U 1
7 L 3
5 D 2
6 D 2
0 R 3

